mypyc/lib-rt/float_ops.c

```diff
@@ -0,0 +1,17 @@
+// Floater point primitive operations
+
+#include <Python.h>
+
+#include "CPy.h"
+
+
+PyObject PyFloat_Add(float left, float right ){
```

Comment:
This function doesn't make much sense: are the arguments floats or PyFloatObjects? The types say one thing, but PyFloat_Check doesn't work on floats, only on PyObjects.

Also, the return type should if anything be a `PyObject *`, not a PyObject.

Reply:
That's exactly where I was having my doubts 
Ints were defined as CPyTagged but there wasn't anything for float and I got confused when I looked at the python C API docs. Can you tell me more problems in my code so I can correct things? And once again,  do I change the PyObject or the float part of the function definition?

Reply:
Also do I have to make an IRBuild test for this? Because I feel I've missed that part as well.

Reply:
Sorry, in this area I know enough to see problems but not enough to come up with solutions. Most likely you'll just have to use boxed floats (PyObjects), since as you said only ints are represented as native objects by mypyc.

Also, I'm pretty sure Python floats are `double`s at the C level, not `float`s.

Reply:
Thank you regardless, I see the error you pointed out, I guess it should be PyFloatObject instead of a float.